lib/core/utils/get-flattened-tree.js

```diff
@@ -86,7 +86,27 @@ function createNode(node, parent, shadowId) {
 }
 
 /**
- * Recursvely returns an array of the virtual DOM nodes at this level
+ * Add children to the parent virtual node
+ * @param {Node[]} childNodes the children of the parent
+ * @param {VirtualNode} parent the parent VirtualNode
+ * @param {String} shadowId, optional ID of the shadow DOM that is the closest shadow
+ *                           ancestor of the node
```

Comment:
Strange formatting - is this whitespace intentional?

Reply:
Just copy pasted from a pervious codeblock, so I guess it must be? I can move it to a single line.